lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -555,13 +558,26 @@ export class PublicClientApplication {
     }
 
     /**
-     * Returns the signed in account
+     * Returns all accounts that MSAL currently has data for.
      * (the account object is created at the time of successful login)
      * or null when no state is found
-     * @returns {@link Account} - the account object stored in MSAL
+     * @returns {@link IAccount[]} - Array of account objects in cache
      */
-    public getAccount(): Account {
-        return this.authModule.getAccount();
+    public getAllAccounts(): IAccount[] {
+        return this.authModule.getAllAccounts();
+    }
+
+    /**
+     * Returns the signed in account matching username.
+     * (the account object is created at the time of successful login)
+     * or null when no state is found
+     * @returns {@link IAccount} - the account object stored in MSAL
+     */
+    public getAccountByUsername(userName: string): IAccount {
+        const allAccounts = this.getAllAccounts();
+        return allAccounts.filter((accountObj) => {
```

Comment:
Nit: this can be a one-liner: `return allAccounts.filter(accountObj => accountObj.username === userName)[0];`

Reply:
done.